godot-codegen/src/class_generator.rs

```diff
@@ -1040,17 +1142,23 @@ fn make_function_definition(
         ctx,
     );
 
+    // TODO We allow(clippy::missing_safety_doc) here, but really we
+    // should figure out the safety requirements of the pointer
+    // functions here and document those somehow. Godot doesn't
+    // document them right now :(
```

Comment:
Maybe we should still add a `# Safety` section, with something like:

> Godot currently does not document safety requirements on this method.
> Make sure you understand the underlying semantics.

This could probably be part of the `safety` variable, with a `#[doc = "..."]` attribute.

Reply:
:+1: